src/doc/trpl/patterns.md

```diff
@@ -70,8 +70,7 @@ This prints `something else`
 
 # Bindings
 
-If you’re matching multiple things, via a `|` or a `...`, you can bind
-the value to a name with `@`:
+You can bind values to names with `@`:
```

Comment:
@steveklabnik I know this is merged but I wonder if you the above is worth addressing separately.